plots.R

```diff
@@ -0,0 +1,121 @@
+# TODO: move to `pacta.multi.loanbook.plot` and export
```

Comment:
I sorta think this should live in `r2dii.plot`? 

Match success rate seems relevant to even a single loanbook? @MonikaFu @jacobvjk thoughts?

Reply:
that is actually where we opened the corresponding issue, see NOTE above. But will update the comment for consistency